### DISCUSSION TITLE
I asked explorer for a review, but it's being pended.

### DISCUSSION THREAD

**Discussion by findmytrueself** [WalletConnect/walletconnect-monorepo#4303] [Developer Support] [answered]
projectId: 4a171d723e54bf3c83f6f74ef9113f6e

Hi, I'm developing a dapp that uses a polygon network and exchanges tokens.

1. I asked explorer for a review, but it's being pended. Can you check it? If there's anything missing, please let me know what I need to fill in.

2. Once the review is completed, I would also like to know if the internal contents can be modified.

3. There are dapp type and wallet type, but after connecting the wallet, should I choose wallet type to request a signature on my personal wallet? I want to know what the difference is.

**Comment by glitch-txs** [comment, marked as answer]
Hi, we are not accepting submissions from Dapps anymore. This won't impact on your integration performance. Thanks!
